src/README.md

```diff
@@ -92,15 +92,17 @@ The XSLT `nist-metaschema-metaprocess.xsl` is a utility XSLT providing a unified
 
 ### Metaschema schemas / `validate` folder
 
-Any metaschema, metaschema module, or composed metaschema, should all be valid to the Metaschema XSD `../../../schema/xml/metaschema.xsd` and to the `validate/metaschema-check.sch` Schematron.
+Any metaschema, metaschema module, or composed metaschema, should all be valid to the Metaschema XSD `../support/metaschema/schema/xml/metaschema.xsd` and to the `validate/metaschema-check.sch` Schematron.
```

Comment:
Needs correction to read: "All metaschema, metaschema module, and composed metaschema, should be valid to the Metaschema XSD" OR "Any metaschema, metaschema module, or composed metaschema, should be valid to the Metaschema XSD"